base/multi.jl

```
@@ -1029,8 +1029,14 @@ function serialize(s::AbstractSerializer, ex::CapturedException)
                                         # if a type used in ex.ex is undefined on the remote node.
 end
 
-original_ex(ex_str) = ErrorException(string("Error deserializing a CapturedException.",
-                                                " Original exception of type : ", ex_str))
+function original_ex(s::AbstractSerializer, ex_str, remote_stktrace)
+    local pid_str = ""
+    try pid_str = string(" from worker ", worker_id_from_socket(s.io)) end
```

Why? I want to ignore any exceptions raised here and it looks pretty clear.